release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3230,3 +3230,19 @@ This is a new AWS permission and you must update credentials for manual mode clu
 ==== Updating
 
 To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-27"]
+=== RHSA-2022:6027 - {product-title} 4.10.27 bug fix and security update
+
+Issued: 2022-08-16
+
+{product-title} release 4.10.27, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:5875[RHSA-2022:6027] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:6026[RHBA-2022:6026] advisory.
```

Comment:
Looks like the URL in the first `link` is still pointing to the older `5875`, but should probably be `6027`?